### DISCUSSION TITLE
app won't update

### DISCUSSION THREAD

**Discussion by bitesh** [bttv-android/bttv#242] [General]
Hello,
So I get prompted to update from 0.10.1-beta to 0.11.0-beta. the bar fills up, then nothing, it just stays there. If I leave, or press back, it prompts me to update again.

**Comment by FoseFx** [comment]
I thought we fixed this in #135. Can you provide me with more information?
- What device are you on (which Android version)?
- Did previous updates work?
- Does this happen every time you try it or only sometimes?
Thank you for reporting this!

**Reply by bitesh** [reply]
it's a Galaxy Samsung A12 running Android 11.
I only found out about this recently, it's the first time I'm updating.
I've tried 3 times already.
Thanks for the prompt response.

**Reply by bitesh** [reply]
Btw, other than that it works great, I'm actually pretty satisfied.

**Reply by FoseFx** [reply]
Do you have anything installed that may prevent the attempt of installing an app, like an antivirus or cleaner app?
Do you have another android device? If so can you try to install [v0.10.1-beta](https://github.com/bttv-android/bttv/releases/tag/v0.10.1-beta) there and attempt an update?

Unfortunately I can't determine the root cause of this without the logs and I am only guessing here. You can always Download the latest update manually here: https://github.com/bttv-android/bttv/releases/latest

**Comment by bitesh** [comment]
so earlier I updated manually. today I tried updating again through the app
and it tells me:
An Error has occurred
Software caused connection to abort
java.net.SocketException: Software caused connection abort
at
java.net.SocketInputStream.socketRead0(Native Method)
at
java.net.SocketInputStream.socketRead(SocketInputStream.java119)
at
java.net.SocketInputStream.read(SocketInputStream.java:176)
at
java.net.SocketInputStream.read(SocketInputStream.java:144)
at
com.android.org.conscrypt.ConscryptEngineSocket$SSLInputStream.readFromSocket(ConscryptEngineSocket.java:945)
at
com.android.org.conscrypt.ConscryptEngineSocket$SSLInputStream.processDataFromSocket(ConscryptEngineSocket.java:909)
at
com.android.org.conscrypt.ConscryptEngineSocket$SSLInputStream.readUntilDataAvailable(ConscryptEngineSocket.java:824)
at
com.android.org.conscrypt.ConscryptEngineSocket$SSLInputStream.read(ConscryptEngineSocket.java:797)
at
okio.InputStreamSource.read(SourceFile:91)
at
okio.AsyncTimeout$source$1.read(SourceFile:129)
at
okio.RealBufferedSource.read(SourceFile:39)
at
okhttp3.internal.http2.Http2Stream$FramingSource.receive$okhttp(SourceFile:441)
at
okhttp3.internal.http2.Http2Stream.receiveData(SourceFile:272)
at
okhttp3.internal.http2.Http2Connection$ReaderRunnable.data(SourceFile:599)
at
okhttp3.internal.http2.Http2Reader.readData(SourceFile:178)
at
okhttp3.internal.http2.Http2Reader.nextFrame(SourceFile:117)
at
okhttp3.internal.http2.Http2Connection$ReaderRunnable.run(SourceFile567)
at
java.lang.Thread.run(Thread.java:923)

Hopefully I copied all that correctly, and it can help.

Regards,
Francisco Reis Pinto

**Comment by bitesh** [comment]
Hello again. Just wanted to mention that this last update (0.12 > 0.13)
worked as intended.
Thank you.

Regards,
Francisco Reis Pinto

**Comment by bitesh** [comment]
Hello once again. This last update from 0.13.1 to 0.14.0, once again gave
what looks a lot to be the same error.

Regards,
Francisco Reis Pinto

**Comment by bitesh** [comment]
Happened again. I just cannot update through the app.

Regards,
Francisco Reis Pinto
